### PR TITLE
EPMRPP-37903: Tooltip added to the filters

### PR DESCRIPTION
**Pull request by tamas-deli-epam**

### CODE REVIEW

**Comment by DiscoElevator** [comment]
Please add Jira task ID to the PR name. (EPMRPP-XXXXX: description)

**Comment by tamas-deli-epam** [comment]
> Please add Jira task ID to the PR name. (EPMRPP-XXXXX: description)

Done

File: app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterTooltipIcon/filterTooltipIcon.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.filter-tooltip-icon {
+  position: relative;
+  top: 2px;
+  display: inline-block;
+  width: 18px;
```
Comment (DiscoElevator) [inline app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterTooltipIcon/filterTooltipIcon.scss]:
```
width: 16px;
height: 16px;
```
as in `app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterItem.scss#25`

Please also move icon svg style from the file above to this file

**Comment by DiscoElevator** [comment]
Closed in favor of #1243

**Pull request closed without merging**